.github/agents/test-development.md

```diff
@@ -122,15 +122,16 @@ Optional language features that the test requires:
 
 ### Behaviors Array
 
-Implementation choices (mutually exclusive pairs):
+Implementation choices (use `conflicts` field to specify incompatible combinations per-test):
 
 | Category | Options | Description |
 |----------|---------|-------------|
 | Boolean parsing | `boolean_strict`, `boolean_lenient` | How to parse boolean values |
 | Line endings | `crlf_preserve_literal`, `crlf_normalize_to_lf` | CRLF handling |
-| Tabs | `tabs_as_content`, `tabs_as_whitespace` | Tab character handling |
-| Spacing | `tabs_as_whitespace`, `tabs_as_content` | Whitespace sensitivity |
+| Tab handling | `tabs_as_content`, `tabs_as_whitespace` | Tab character treatment in values |
+| Indentation | `indent_spaces`, `indent_tabs` | Output indentation style |
```

Comment:
Clarify that this is output/printed indentation, not about parsing. Actually maybe we can call out the functions that are affected by each behavior, inferred from the test data?